src/main/java/org/fundacionjala/coding/sergio/FizzBuzz.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.coding.sergio;
+
+/**
+ * Created by SergioNavarro on 8/25/2017.
+ */
+public class FizzBuzz {
+
+    /**
+     * @param num input.
+     * @return return.
+     */
+    public String[] resolve(int num) {
+        String[] result = new String[num];
+        boolean normal = true;
```

Comment:
Do we really need this flag ?
I don't think so
Please change the logic and remove it.